_posts/2022-02-22-scala-free.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: "Scala Free in One Fifteen"
+date: 2022-02-22 00:00:00
+authors:
+- sjwiesman:
+  name: "Seth Wiesman"
+  twitter: "sjwiesman"
+excerpt: excerpt: Apache Flink's runtime is now Scala free, allowing users to leverage any Scala version in their user code - including Scala 3!
+---
+
+Flink 1.15 is right around the corner, and among the many improvements is a Scala free classpath by default.
```

Comment:
nuh-uh. Scala is still on the CP by default; users must opt-in to Scala-free Flink by removing the flink-scala.jar from lib/.
See also the release notes in https://issues.apache.org/jira/browse/FLINK-13414.